code/modules/reagents/chemistry/reagents/alcohol_reagents.dm

```diff
@@ -91,7 +91,7 @@ All effects don't start immediately, but rather get worse over time; the rate is
 	color = "#664300" // rgb: 102, 67, 0
 	nutriment_factor = 1 * REAGENTS_METABOLISM
 	boozepwr = 25
-	taste_description = "piss water"
```

Comment:
what in the god damn

Reply:
> what in the god damn

yeah the beer flavor description has been "piss water" since its inception, this PR incidentally changes that, thank uhhhh @ErdinyoBarboza